lib/assets/sql/tdr_indexes/query_w_analysis_file.sql

```diff
@@ -29,6 +29,19 @@ https://docs.google.com/document/d/1KJcF6aWLub--yvTIeZkd_VgTBChinuFqK2aCzSo6hgQ/
 Swagger docs for TDR-dev API
 https://jade.datarepo-dev.broadinstitute.org/swagger-ui.html#/
 
+Changes:
+previous SQL extracted ".text" values, now using ".ontology_label" values
```

Comment:
These comments seem helpful overall, but "Changes" strike me as better for a commit message, or at least worth noting "from" (before) and "as of" dates.  I don't feel strongly about it, though -- if you think it's an overall benefit for maintainability let's keep it.

Reply:
Good point, removed trivial change info, kept design decisions.